Cargo.Bazel.Fuzzing.json.lock

```diff
@@ -13243,6 +13243,61 @@
       ],
       "license_file": "LICENSE"
     },
+    "core2 0.4.0": {
```

Comment:
did you mean to cause so many updates to cargo files?

Reply:
These must be transitive dependencies.